.circleci/workflows.yml

```diff
@@ -0,0 +1,771 @@
+            # Workaround for sharing this variable to the next steps
+            echo "export TAG=$TAG" >> $BASH_ENV
+            echo "Docker images will be tagged with: ${TAG}"
+            echo $TAG > /tmp/docker-tag.txt
+      - run:
+          name: Build & Publish Management API Docker Image to Azure Registry
+          command: |
+            export AM_VERSION=$(mvn -s /tmp/.circleci.settings.xml -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec)
+            echo "Gravitee AM version: ${AM_VERSION}"
+            export IMAGE_TAG=graviteeio.azurecr.io/am-management-api:${TAG}
+            echo "export IMAGE_TAG=$IMAGE_TAG" >> $BASH_ENV
+            docker build -f docker/management-api/Dockerfile-dev --build-arg GRAVITEEAM_VERSION=${AM_VERSION} -t ${IMAGE_TAG} /tmp
+      - snyk/scan:
+          severity-threshold: high
+          docker-image-name: ${IMAGE_TAG}
+          fail-on-issues: true
+      - run:
+          name: Push API Docker image
+          command: |
+            echo $AZURE_DOCKER_REGISTRY_PASSWORD | docker login --username $AZURE_DOCKER_REGISTRY_USERNAME --password-stdin graviteeio.azurecr.io
+            docker push ${IMAGE_TAG}
+      - run:
+          name: Build & Publish Gateway Docker Image to Azure Registry
+          command: |
+            export AM_VERSION=$(mvn -s /tmp/.circleci.settings.xml -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec)
+            export IMAGE_TAG=graviteeio.azurecr.io/am-gateway:${TAG}
+            echo "export IMAGE_TAG=$IMAGE_TAG" >> $BASH_ENV            
+            docker build -f docker/gateway/Dockerfile-dev --build-arg GRAVITEEAM_VERSION=${AM_VERSION} -t ${IMAGE_TAG} /tmp
+      - snyk/scan:
+          severity-threshold: high
+          docker-image-name: ${IMAGE_TAG}
+          fail-on-issues: true
+      - run:
+          name: Push Gateway Docker image
+          command: |           
+            echo $AZURE_DOCKER_REGISTRY_PASSWORD | docker login --username $AZURE_DOCKER_REGISTRY_USERNAME --password-stdin graviteeio.azurecr.io
+            docker push ${IMAGE_TAG}
+      - run:
+          name: Build & Publish Web UI Docker Image to Azure Registry
+          command: |
+            cp -fr docker/management-ui/config /tmp/
+            cp -fr docker/management-ui/run.sh /tmp/
+
+            export AM_VERSION=$(mvn -s /tmp/.circleci.settings.xml -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec)
+            export IMAGE_TAG=graviteeio.azurecr.io/am-management-ui:${TAG}
+            echo "export IMAGE_TAG=$IMAGE_TAG" >> $BASH_ENV            
+            docker build -f docker/management-ui/Dockerfile-dev --build-arg GRAVITEEAM_VERSION=${AM_VERSION} -t ${IMAGE_TAG} /tmp
+      - snyk/scan:
+          severity-threshold: high
+          docker-image-name: ${IMAGE_TAG}
+          fail-on-issues: true
```

Comment:
No sure that we should stop le pipeline on error in this job as it will prevent deploying our images on nightly.
the scan at this stage is just to inform us that we are safe on docker images and if not we will be able to do whatever it takes to fix the issue before releasing AM. (same comment for the two other images)

what is your thought about this point?

Reply:
I agree Eric.  we can continue pipeline by setting this parameter fail-on-issues to false

Ref : https://circleci.com/developer/orbs/orb/snyk/snyk#usage-quickstart-docker-scanning

https://circleci.com/blog/security-with-snyk-in-the-circleci-workflow/